root/static/scripts/common/components/ArtistCreditLink.js

```diff
@@ -47,6 +47,7 @@ export const MpIcon = (hydrate<MpIconProps>('span.ac-mp', (
 
   return (
     <Tooltip
+      ContainerElement="span"
```

Comment:
Is there any situation where we *need* divs, and spans with inline-block wouldn't work?

(unrelated, but I also find it strange to capitalize these variables, which don't refer to classes or components - they are just storing the string name of an element)